ko/news/_posts/2018-11-29-fukuoka-ruby-award-2019.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: news_post
+title: "2018 후쿠오카 루비 경진대회 - Matz가 심사합니다"
+author: "Fukuoka Ruby"
+translator: "shia"
+date: 2018-11-29 00:00:00 +0000
+lang: ko
+---
+
+루비스트 여러분에게,
+
+일본 후쿠오카 현은 'Matz' 마츠모토 씨와 함께 여러분을 다음 경진 대회에 초대합니다. 흥미로운 루비 프로그램을 개발하셨다면 한번 지원해보세요.
+
+2018 후쿠오카 루비 경진대회 - 대상 - 백만 엔!
+
+접수 마감: 2019년 1월 31일
+
+![Fukuoka Ruby Award](http://www.digitalfukuoka.jp/javascripts/kcfinder/upload/images/fukuokarubyaward2017.png)
+
+Matz를 포함한 패널들이 후쿠오카 경진대회의 우승자를 선택합니다. 후쿠오카 경진대회의 대상에게는 백만 엔의 상금이 주어집니다. 이전 이 대회의 우승자로는 Rhomobile(미국)과 APEC 기후 센터(한국)가 있습니다.
+
+<http://myfukuoka.com/category/news/ruby-news/>
+
+대회에 참가하는 프로그램은 완전히 루비로 작성될 필요는 없습니다. 하지만 루비의 특징을 살려야 합니다.
+
+프로젝트는 지난 12개월 동안 개발되거나 완료된 것만이 유효합니다. 추가 설명이나 참가를 원하신다면 다음 웹사이트를 방문하세요.
+
+[http://www.digitalfukuoka.jp/events/185](http://www.digitalfukuoka.jp/events/185)
+또는
+[http://myfukuoka.com/events/2019-fukuoka-ruby-award-guidelines-for-applicants/](http://myfukuoka.com/events/2019-fukuoka-ruby-award-guidelines-for-applicants/)
+
+[http://www.digitalfukuoka.jp/uploads/event_detail/file/465/RubyAward_ApplicationForm_2019.doc](http://www.digitalfukuoka.jp/uploads/event_detail/file/465/RubyAward_ApplicationForm_2019.doc)
+
+지원서는 award@f-ruby.com으로 보내주세요.
+
+올해에는 다음과 같은 특별상이 준비되어 있습니다.
+
+AWS 상의 수상자는 다음을 받게 됩니다.
+
+* 아마존 에코 (변경될 가능성 있음)
+* AWS 아키텍트의 기술 상담
+
+GMO 페파보 상의 수상자는 다음을 받게 됩니다.
+
+* Lolipop! 공유 호스팅 서비스: 기본 플랜의 10년 구독권이나 클라우드 서비스에서 사용 가능한 100,000엔 분의 쿠폰
+* Muumuu 도메인 DNS 등록 서비스: 1개 도메인의 10년 무료 구독권(매년 10,000엔 이하의 비용이 드는 도메인에 한하여)
+
+IIJ GIO 상의 수상자는 다음을 받게 됩니다.
+
+* IIJ GIO 500,000엔의 무료 쿠폰(6개월간 유효)
+
+Money Forward 상의 수상자는 다음을 받게 됩니다.
+
+* Money Forward에서 일하고 있는 루비 커미터와의 저녁 식사
+* 개인 재정 관리 서비스 'Money Forward'의 10년 프리미엄 무료 이용권
+
+Salesforce 상의 수상자는 다음을 받게 됩니다.
+
+* salesforce.com의 기념품
+
+'Matz는 코드를 주의깊게 테스트하고 읽어볼 것이므로, 지원할만한 충분한 가치가 있을 것입니다! 대회에 참가하는 것은 무료입니다.'
```

Comment:
```suggestion
'Matz'는 코드를 주의깊게 테스트하고 읽어볼 것이므로, 지원할만한 충분한 가치가 있을 것입니다! 대회참가는 무료입니다.'
```

Reply:
I think surrounding this line with `"` would be more appropriate than `'`. How do you think?

Reply:
I followed this one, https://github.com/ruby/www.ruby-lang.org/pull/1872#discussion_r231775969
Is there any reason you suggest to use `"` here?